kit/src/layout/chatbar/mod.rs

```diff
@@ -157,7 +197,48 @@ pub fn Chatbar<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
             div {
                 class: "controls",
                 cx.props.controls.as_ref()
-            }
+            },
+            (!cx.props.emoji_suggestions.is_empty()).then(|| rsx!(EmojiSuggesions {
+                suggestions: cx.props.emoji_suggestions,
+                on_emoji_click: move |(emoji, alias)| {
+                    if let Some(e) = cx.props.on_emoji_click.as_ref() {
+                        e.call((emoji, alias, cursor_position.read().unwrap()))
```

Comment:
remove the call to `unwrap()`. `kit` shouldn't panic if  the code using `Chatbar` doesn't correctly call `oncursor_update` before calling `on_emoji_click`